harmony_netcdf_to_zarr/convert.py

```diff
@@ -159,9 +159,9 @@ def _output_worker(output_queue: Queue, shared_namespace: Namespace,
             with Dataset(input_granule, 'r') as input_dataset:
                 input_dataset.set_auto_maskandscale(False)
                 __copy_group(input_dataset,
-                             create_zarr_group(zarr_store, overwrite=False),
-                             zarr_synchronizer, dim_mapping,
-                             aggregated_dimensions)
+                             create_zarr_group(zarr_store, overwrite=False,
+                                               synchronizer=zarr_synchronizer),
```

Comment:
Adding the synchronizer here solved the biggest issue with copying group metadata attributes.